src/uikit/Wizard.js

```diff
@@ -79,9 +80,12 @@ export default compose(
       setIndex(index + 1 >= steps.length ? index : index + 1),
     prevStep: ({ index, setIndex, steps }) => event => setIndex(index - 1 < 0 ? index : index - 1),
   }),
-  withPropsOnChange(['index'], ({ index, setIndex, steps }) => ({
-    currentStep: steps[index] || { title: 'no step', Component: '--' },
-  })),
+  withPropsOnChange(['index'], ({ index, setIndex, steps }) => {
```

Comment:
`react-router` has `withRouter`, which populates props w/ `{ match, location, history }`, which is preferred over `window.location`